dataprep/eda/utils.py

```diff
@@ -435,12 +435,13 @@ def _format_axis(fig: Figure, minv: int, maxv: int, axis: str) -> None:
     """
     Format the axis ticks
     """  # pylint: disable=too-many-locals
+
     # divisor for 5 ticks (5 results in ticks that are too close together)
     divisor = 4.5
     # interval
     gap = (maxv - minv) / divisor
     # get exponent from scientific notation
-    _, after = f"{gap:.0e}".split("e")
+    _, after = f"{gap:.0e}".split("e") if gap != np.inf else ("", "0")
```

Comment:
Could we use `np.isinf` to test inf? It also covers other cases such as `-np.inf`.